### PR TITLE
Fixes nil tokens for updated subscription payment sources

### PR DESCRIPTION
**Pull request by LukeSettle**
Fixes nil tokens and updates rubocop to 2.5.

### CODE REVIEW

File: app/models/solidus_subscriptions/subscription.rb [hunk 1]
```diff
@@ -228,5 +230,10 @@ def check_total_skips_exceeded
     def line_item
       line_items.first
     end
+
+    def create_payment_method_profile
+      payment = OpenStruct.new(source: self.wallet_payment_source.payment_source)
+      payment.source.payment_method.create_profile(payment)
```
Comment (fastjames) [inline app/models/solidus_subscriptions/subscription.rb:239]: Is this code seen somewhere else, maybe in `solidus_paypay_braintree`?

Comment (LukeSettle) [reply app/models/solidus_subscriptions/subscription.rb:239]: This is from solidus in a payment `after_save` method https://github.com/geminimvp/solidus/blob/master/core/app/models/spree/payment.rb#L234

Comment (fastjames) [reply app/models/solidus_subscriptions/subscription.rb:239]: OK, can you verify that this fills in the `token` attribute that we need? This sounds tough to put in a spec.

Comment (LukeSettle) [reply app/models/solidus_subscriptions/subscription.rb:239]: Here is a source that was updated from subscription update with the token included:

image

File: app/models/solidus_subscriptions/installment.rb [hunk 2]
```diff
@@ -120,7 +120,8 @@ def advance_actionable_date!
 
     def next_actionable_date
       return if Config.reprocessing_interval.nil?
-      (DateTime.current + Config.reprocessing_interval).beginning_of_minute
+
+      (Time.current + Config.reprocessing_interval).beginning_of_minute
```
Comment (fastjames) [inline app/models/solidus_subscriptions/installment.rb:124]: is `Time.current` timezone aware?

**Review by fastjames — APPROVED** [review]
:+1:

File: app/models/solidus_subscriptions/installment.rb (hunk 2 above)
Comment (fastjames) [reply app/models/solidus_subscriptions/installment.rb:124]: Aha, Time.current calls `Time.zone.now`.

**Review by cnorm35 — APPROVED** [review]

**Pull request merged**